app/models/config_report.rb

```diff
@@ -0,0 +1,20 @@
+class ConfigReport < Report
+  def self.model_name
+    superclass.model_name
+  end
+
+  def self.import(report, proxy_id = nil)
+    ConfigReportImporter.import(report, proxy_id)
+  end
+
+  def summary_status
+    return _("Failed")   if error?
+    return _("Modified") if changes?
+    _("Success")
+  end
+
+  # puppet report status table column name
+  def self.report_status
```

Comment:
Nitpick, perhaps call this report_status_column so it's more different to ReportImporter.report_status.